cmp/chart/Chart.ts

```diff
@@ -85,6 +86,7 @@ export const [Chart, chart] = hoistCmp.withFactory<ChartProps>({
         const coreContents = box({
             ...layoutProps,
```

Comment:
See above comment. If we had a `getBoxProps` method, we could just spread the `boxProps` here and let `box` handle the `TestSupportProps` (presumably if we introduce more than just `testId`, that could come in handy, though I'm not sure we would)